interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/use-report-parameters/parameters-overview.md

```diff
@@ -4,75 +4,95 @@ author: Anna Gubareva
+		Specify the **Sort Member** and **Sort Order** properties to sort parameter values.
 		
 	* **Static values**
 		
-		Switch to this tab to specify a static list of possible values. Each value should have a description that is displayed in a Print Preview.
+		Switch to this tab to specify a static list of values. Each value should have a description that is displayed in Print Preview.
 		
 		![](../../../../../images/eurd-win-parameters-static-values.png)
 
-* **Allow multiple values** (corresponds to the parameter's **Multi-Value** property) - when this option is enabled, a parameter can be assigned a [collection of values](create-multi-value-and-cascading-parameters.md).
+## Use Parameters
 
-* **Allow null value** (corresponds to the parameter's **Allow Null** property) - indicates whether the parameter's value can be unspecified.
+Report parameters pass data to a report before it is generated in Print Preview. You can specify parameter values.
 
-## <a name="useparameters"></a>Use Parameters
-You can use report parameters to solve the following tasks:
+Use report parameters in the following cases:
+
+* **Filter**
+
+	Parameters can provide values to a report's **Filter String** to [filter report data](../filter-data/filter-data-at-the-report-level.md).
 
-* **Filter Data**
-	
-	When [filtering report data](../filter-data/filter-data-at-the-report-level.md), parameters can be used for providing values to a report's **Filter String** property.
-	
 	![](../../../../../images/eurd-win-parameters-in-filter-string.png)
-	
-	When [filtering data at the level of a data source](../filter-data/filter-data-at-the-data-source-level.md), you can link report parameter to [query parameters](use-query-parameters.md) that are used in the SELECT statement of a SQL string.
+
+	You can link the report parameter to [query parameters](use-query-parameters.md) used in the SQL string's SELECT statement to [filter data at the data source level](../filter-data/filter-data-at-the-data-source-level.md).
 
 * **Bind to Data**
 	
-	You can bind a report control to a parameter and display its value in the report. To create a new label bound to a parameter, drag the parameter from the [Field List](../../report-designer-tools/ui-panels/field-list.md) and drop it onto the required band.
+	You can bind a report control to a parameter and display its value in a report. To create a new [label](../../use-report-elements/use-basic-report-controls/label.md) bound to a parameter, drag the parameter from the [Field List](../../report-designer-tools/ui-panels/field-list.md) and drop it onto a band.
 	
 	![](../../../../../images/eurd-win-parameters-for-data-binding.png)
 	
-	When using [mail merge](../../bind-to-data/use-embedded-fields-mail-merge.md), you can refer to a parameter by adding a question mark before its name.
+	You can refer to a parameter in [mail merge](../../bind-to-data/use-embedded-fields-mail-merge.md). Add a question mark in front of the parameter's name.
```

Comment:
You can... You can... You can... You can... You are creating a pattern here.

Reply:
These sentences can be combined.

Reply:
Ok